gethoney/main.py

```diff
@@ -1,49 +1,51 @@
-from fastapi import FastAPI
-from gethoney.models import Honeypot
-from gethoney.crud import conn, curr
-import requests
-
-app = FastAPI()
-
-curr.execute(
-    """CREATE TABLE IF NOT EXISTS honeypots
-    (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT, url TEXT, description TEXT)"""
-)
-
-# curr.execute(
-#     """CREATE TABLE IF NOT EXISTS logs
-#     (id INTEGER PRIMARY KEY, log_id INTEGER, name TEXT))"""
-# )
+from fastapi import Depends, FastAPI, HTTPException
 
+from gethoney.crud import Database
+from gethoney.models import Honeypot
 
-@app.get("/")
-def read_root():
-    return {"Hello": "World"}
+db_path = "../data/gethoney.db"
 
+app = FastAPI()
 
-@app.post("/honeypots/", status_code=201)
-def create_honeypots(honeypots: list[Honeypot]):
 
-    curr.executemany(
-        """INSERT INTO honeypots
-                (name, url, description)
-                VALUES (?, ?, ?) """,
-        [(honeypot.name, honeypot.url, honeypot.description) for honeypot in honeypots],
-    )
-    conn.commit()
-    return honeypots
+def get_db():
+    yield Database(db_path)
 
 
-@app.get("/honeypots/")
-def get_honeypots():
-    curr.execute("SELECT * FROM honeypots")
-    data = curr.fetchall()
-    return data
+@app.get("/honeypots/{id}", status_code=200)

```

Comment:
All of your functions here are missing return value types, ex.
`def my_func() -> None:`

Should probably double check all your functions for correct typings